services/api/service.go

```diff
@@ -493,6 +527,85 @@ func (api *RelayAPI) startValidatorRegistrationDBProcessor() {
 	}
 }
 
+// simulateBlock sends a request for a block simulation to blockSimRateLimiter.
+func (api *RelayAPI) simulateBlock(ctx context.Context, opts blockSimOptions) (requestErr, validationErr error) {
+	t := time.Now()
+	requestErr, validationErr = api.blockSimRateLimiter.Send(ctx, opts.req, opts.isHighPrio, opts.fastTrack)
+	log := opts.log.WithFields(logrus.Fields{
+		"duration":   time.Since(t).Seconds(),
+		"numWaiting": api.blockSimRateLimiter.CurrentCounter(),
+	})
+	if validationErr != nil {
+		ignoreError := validationErr.Error() == ErrBlockAlreadyKnown || validationErr.Error() == ErrBlockRequiresReorg || strings.Contains(validationErr.Error(), ErrMissingTrieNode)
+		if !ignoreError {
+			log.WithError(validationErr).Warn("block validation failed")
+			return nil, validationErr
+		}
+		log.WithError(validationErr).Warn("ignorable error during block simulation")
```

Comment:
now there's a return after this, maybe we stick with the pattern of the other logs here?

Also, I'd tend towards making all these messages info level rather than warn. But that's not a biggie or needed rn

```suggestion
		log.WithError(validationErr).Warn("block validation failed with ignorable error")
```

Reply:
changed the message! left as warning for now. i still find the warning filtering useful :-)